src/components/UserSelector.tsx

```diff
@@ -1,19 +1,65 @@
-import React from 'react';
+import {
+  FC,
+  useEffect,
+  useState,
+  SetStateAction,
+} from 'react';
+import cn from 'classnames';
+import { User } from '../types/User';
+import { Error } from '../types/Error';
+import { client } from '../utils/fetchClient';
+
+type Props = {
+  selectedUser: User | null,
+  setSelectedUser: (user: User) => void,
+  setError: (err: Error) => void,
+  setIsLoader: (arg: boolean) => void,
+};
+
+export const UserSelector: FC<Props> = ({
+  selectedUser,
+  setSelectedUser,
+  setError,
+  setIsLoader,
+}) => {
+  const [isDropdown, setIsDropdown] = useState(false);
+  const [users, setUsers] = useState<User[]>([]);
+
+  useEffect(() => {
+    client.get<SetStateAction<User[]>>('/users')
```

Comment:
The type used in client.get<SetStateAction<User[]>>('/users') is incorrect. Replace `SetStateAction<User[]>` with `User[]` to fix the issue. 😊